content/en/observability_pipelines/quickstart/_index.md

```diff
@@ -0,0 +1,170 @@
+{{% /tab %}}
+{{% tab "Azure AKS" %}}
+On Azure, we prefer to use `default` (which really just means `managed-csi`) disks as their performance is satisfactory for our needs.
+{{% /tab %}}
+{{% tab "Google GKE" %}}
+On GKE, we recommend using the `premium-rwo` drive class, as that is backed by SSDs. **Using the HDD-backed class, `standard-rwo`, will likely not give you enough write performance for the buffers to be useful, in our estimation.**
+{{% /tab %}}
+{{< /tabs >}}
+
+## Linking the Agent and the Worker
+To send logs and metrics to the Observability Pipelines Worker, update your agent configuration with the following:
+
+```yaml
+vector:
+  logs.enabled: true
+  # Adjust protocol to https if TLS/SSL is enabled on the Observability Pipelines Worker
+  logs.url: "http://<OPW_HOST>:8282"
+  metrics.enabled: true
+  # Adjust protocol to https if TLS/SSL is enabled on the Observability Pipelines Worker
+  metrics.url: "http://<OPW_HOST>:8282"
+
+```
+
+`OPW_HOST` will be the `EXTERNAL-IP` of the load balancer you set up earlier, which you can retrieve by running the following command:
+
+```
+$ kubectl get svc opw-observability-pipelines-worker
+```
+
+At this point, your observability data should be flowing through the OP Worker, and is now available for whatever processing you wish to do to it. In the next section, we'll cover both what processing we include by default, as well as additional options that are available to you.
+
+## Working with Data
+The Helm charts provided above give some example processing steps both to demonstrate the power of OP's tools, and to ensure that data sent to Datadog is in a correct format. We'll go over these inclusions below.
+
+### Processing Logs
+The provided logs pipeline has a couple of distinct purposes:
+
+- **Tag logs coming through the OP Worker.** This will help with determining what traffic still needs to be shifted over to the Worker, as you update your clusters. These tags will also let you see how logs are being routed through the load balancer, in case of imbalances.
+- **Correct the status of logs coming through the Worker.** Due to a quirk in how the Datadog Agent collects logs from containers, the provided `.status` attribute does not properly reflect the true level of the message. We remove this in order to prevent some issues with parsing rules in the backend that receives logs from the Worker.
+
+In particular, the following steps in the pipeline are worth calling out:
+- `logs_parse_ddtags`
+- `logs_finish_ddtags`
+
+Internally, the Datadog Agent represents log tags as a CSV in a single string. In order to effectively manipulate these tags, we must parse them, perform our modifications, and then re-encode them before they are sent off to the ingest endpoint. These steps are written to automatically perform those actions for you. **Any modifications you make to the pipeline, especially for manipulating tags, should be sandwiched between these two steps.**
```

Comment:
```suggestion
Internally, the Datadog Agent represents log tags as a CSV in a single string. To effectively manipulate these tags, they must be parsed, modified , and then re-encoded before they are sent to the ingest endpoint. These steps are written to automatically perform those actions for you. Any modifications you make to the pipeline, especially for manipulating tags, should be in between these two steps.
```

Reply:
In between which two steps?

Reply:
In between the `logs_parse_ddtags` and `logs_finish_ddtags` steps.